cita-auth/src/block_verify.rs

```diff
@@ -71,6 +75,26 @@ impl<'a> BlockVerify<'a> {
         true
     }
 
+    // min quota is 21000 or (32000 if tx.to == 0) + (4 per byte)
+    // tx quota: 21000
+    // create contract: 32000
+    fn verify_min_quota(&self, sig_tx: &SignedTransaction) -> bool {
+        let tx = sig_tx.get_transaction_with_sig().get_transaction();
+        let tx_version = tx.get_version();
+        if tx_version < 3 {
```

Comment:
It always return true. how to verify min quota?

Reply:
only verify tx with new tx_version

Reply:
Get it.
Actually, it's no need to consider the version at `cita-auth` module

Reply:
I think it's needed for tx in early blocks.